.travis.yml

```diff
@@ -4,13 +4,12 @@ language: java
 before_install:
   - mkdir ci-dependencies
   - cd ci-dependencies
-  - curl -O https://openskywalking.github.io/skywalking-ci-assist/jars/ojdbc14-10.2.0.4.0.jar
+  - curl -O https://github.com/SkyAPM/ci-assist/blob/master/jars/ojdbc14-10.2.0.4.0.jar
   - mvn install:install-file  -Dfile=ojdbc14-10.2.0.4.0.jar  -DgroupId=com.oracle  -DartifactId=ojdbc14 -Dversion=10.2.0.4.0 -Dpackaging=jar
-  - curl -O https://openskywalking.github.io/skywalking-ci-assist/jars/resin-4.0.41.jar
+  - curl -O https://github.com/SkyAPM/ci-assist/blob/master/jars/resin-4.0.41.jar
```

Comment:
Same pattern here.

Reply:
As you mentioned , I changed the uri on both ojdbc and resin.